### PR TITLE
Text update of PIM EE archive installation

### PR DESCRIPTION
**Pull request by pandino101**

### CODE REVIEW

File: install_pim/partners_portal_archive.rst.inc [hunk 1]
```diff
@@ -1,34 +1,72 @@
-Getting Akeneo Partners Portal access
+Getting Akeneo Partners Portal Access
 *************************************
-After obtaining your Enterprise Edition license you should have recieved an access to the Akeneo Partners Portal.
+Akeneo Partners Portal login credentials are sent to you by email. If case you haven't recieved it you can request one at: http://helpdesk.akeneo.com/
```
Comment (rybus) [inline install_pim/partners_portal_archive.rst.inc]: If you haven't [...], you can request one [...]

**Comment by nidup** [comment]
a pull request on master branch means it will be visible only on the upcoming version of the public doc, if you want to make it appears on the current default doc (https://docs.akeneo.com/latest/index.html) the PR must be open on 1.7 branch :)

**Comment by pandino101** [comment]
@nidup Can I update my commits to be on 1.7?
If so, will these changes be integrated into the Master also? or should I make 2 PR for both versions?

**Comment by nidup** [comment]
when we do a pull request on a version, then the core team apply these changes (merge) on upcoming version. if you do on 1.7 we'll apply on master (upcoming version) too. i take a look how we can pick your changes to apply it on 1.7 and i let you know :)

File: install_pim/partners_portal_archive.rst.inc (hunk 1 above)
Comment (nidup) [inline install_pim/partners_portal_archive.rst.inc]: i'll fix it until the merge done :)

**Comment by nidup** [comment]
congrats @pandino101 for your first pull request on the documentation, I applied it on 1.7 too. you can claim this badge http://badger.akeneo.com/badge/0a25c62e-1bae-11e6-92dc-d60437e930cf

thank you!

PS: you can get in touch with @LaureBro to know when the Squirrel squad plan to deploy the doc

**Comment by pandino101** [comment]
Thanks!

**Pull request merged**